app/src/main/kotlin/com/camerasecuritysystem/client/SettingsActivity.kt

```diff
@@ -15,50 +16,53 @@ import kotlinx.coroutines.launch
 class SettingsActivity : AppCompatActivity(),
     ConnectDialog.ConnectDialogListener {
 
-    private var binding: ActivitySettingsBinding? = null
+    private lateinit var binding: ActivitySettingsBinding
 
     private var keyStore = KeyStoreHelper("connectToServer")
 
-    lateinit var connectionLiveData: ConnectionLiveData
+    private var isFragmentDurationValid: Boolean = false
 
-    lateinit var connectBtn : Button
+    lateinit var connectionLiveData: ConnectionLiveData
```

Comment:
Can you change these around? That way, the connectionLiveData won't be seen as a change. This in order to minimize merge conflicts later.

Reply:
Done